Signum.React/Scripts/Constructor.tsx

```diff
@@ -75,7 +76,7 @@ export function constructPack(type: string | Type<any>, props?: any, pr?: Proper
 
   assertCorrect(result);
 
-  return Navigator.toEntityPack(result);
+  return API.fetchEntityPackEntity(result as Entity);

```

Comment:
Why you did this? if result is a an EmbeddedEntity or a ModelEntity calling the server will be unnecessary or problematic.

Reply:
Our assumption is, that constructing an `Entity` from the UI must always initialize an Entity in the same way as if it would have been constructed in the backend. Therefore the idea is to use the entity that is returned with the entity pack instead of the empty object that is initialized in the frontend.

If I'm not mistaken, this was also the behavior in older versions of the framework, wasn't it?